css/src/metabox.scss

```diff
@@ -45,8 +45,9 @@ ul.wpseo-metabox-tabs:after {
 	clear: both;
 }
 
-.wpseo-metabox-tabs-div ul {
+#wpseo_meta.wpseo-metabox .wpseo-metabox-content ul {
 	list-style: none;
+	list-style-type: none;
```

Comment:
These two declarations do the same thing: the properties `list-style` and `list-style-type` are equivalent, please use just one of them.

Worth considering: in https://github.com/Yoast/wordpress-seo/pull/5322 we've added a `yoast` CSS class as a "root" class for the plugin content. That is, if we're consistent in using it, we can just use `.yoast` to target the plugin content and increase CSS rules specificity when needed. I'd consider to use
`.yoast.wpseo-metabox .wpseo-metabox-content ul` instead of the ID selector `#wpseo_meta`